proto/rill/admin/v1/api.proto

```diff
@@ -606,6 +637,40 @@ message UpdateOrganizationResponse {
   Organization organization = 1;
 }
 
+message UpdateOrganizationBillingPlanRequest {
+  string org_name = 1;
+  optional string plan_name = 2;
+  optional string biller_plan_id = 3;
```

Comment:
Can we decide on just one way of doing this?

Reply:
moved to using plan name only as its a more friendly option for cli but also enforces that all plans defined in Orb have an external plan id defined.